data/dbus/com.canonical.SafeLauncher.service.in

```diff
@@ -0,0 +1,3 @@
+[D-BUS Service]
+Name=com.canonical.SafeLauncher
```

Comment:
I saw there are some discussions around having something io.snapcraft.* based. I am fine with that and we can introduce this as a variant in another PR and migrate to it over time.